pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1131 @@
+	}
+}
+
+func (GitRepoVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":           "GitRepoVolumeSource represents a volume that is pulled from git when the pod is created.",
+		"repository": "Repository URL",
+		"revision":   "Commit hash for the specified revision. Optional.",
+	}
+}
+
+func (GlusterfsVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "GlusterfsVolumeSource represents a Glusterfs Mount that lasts the lifetime of a pod.",
+		"endpoints": "EndpointsName is the endpoint name that details Glusterfs topology. Required. See http://releases.k8s.io/HEAD/examples/glusterfs/README.md#create-a-pod",
+		"path":      "Path is the Glusterfs volume path. Required. See http://releases.k8s.io/HEAD/examples/glusterfs/README.md#create-a-pod",
+		"readOnly":  "ReadOnly here will force the Glusterfs volume to be mounted with read-only permissions. Defaults to false. Optional. See http://releases.k8s.io/HEAD/examples/glusterfs/README.md#create-a-pod",
+	}
+}
+
+func (HTTPGetAction) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":       "HTTPGetAction describes an action based on HTTP Get requests.",
+		"path":   "Path to access on the HTTP server. Optional.",
+		"port":   "Name or number of the port to access on the container. Number must be in the range 1 to 65535. Name must be an IANA_SVC_NAME. Required.",
+		"host":   "Host name to connect to, defaults to the pod IP. Optional.",
+		"scheme": "Scheme to use for connecting to the host. Defaults to HTTP. Optional.",
+	}
+}
+
+func (Handler) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "Handler defines a specific action that should be taken",
+		"exec":      "One and only one of the following should be specified. Exec specifies the action to take.",
+		"httpGet":   "HTTPGet specifies the http request to perform.",
+		"tcpSocket": "TCPSocket specifies an action involving a TCP port. TCP hooks not yet supported",
+	}
+}
+
+func (HostPathVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":     "HostPathVolumeSource represents bare host directory volume.",
+		"path": "Path of the directory on the host. See http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#hostpath",
+	}
+}
+
+func (ISCSIVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":             "ISCSIVolumeSource describes an ISCSI Disk can only be mounted as read/write once.",
+		"targetPortal": "iSCSI target portal. The portal is either an IP or ip_addr:port if the port is other than default (typically TCP ports 860 and 3260). Required.",
+		"iqn":          "Target iSCSI Qualified Name. Required.",
+		"lun":          "iSCSI target lun number. Required.",
+		"fsType":       "Filesystem type to mount. Must be a filesystem type supported by the host operating system. Ex. \"ext4\", \"xfs\", \"ntfs\" Required.",
```

Comment:
"Required: Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#iscsi"